packages/kit/src/utils/url.spec.js

```diff
@@ -2,6 +2,9 @@ import * as assert from 'uvu/assert';
 import { describe } from './unit_test.js';
 import { resolve, normalize_path, make_trackable, disable_search } from './url.js';
 
+// @ts-expect-error define global required in url.js
+globalThis.__SVELTEKIT_BROWSER__ = false;
```

Comment:
didn't find a better way to achieve this. Using uvu's `-r somefile.js` doesn't work because that somehow switches everything to using `require`, which fails.